src/arco_era5/pangeo.py

```diff
@@ -112,6 +112,8 @@ def parse_args(desc: str, default_chunks: t.List[str]) -> t.Tuple[argparse.Names
     parser.add_argument('-c', '--chunks', metavar='chunks', nargs='+',
                         default=default_chunks,
                         help='Chunks of variables to merge together.')
+    parser.add_argument('-t', '--target-chunk', type=json.loads, default='{"time": 1}',
```

Comment:
This LGTM! One last request: please update the model-level script to make use of this flag and chunk by level and time.

Reply:
To clarify the above: Can you update the how-to-run examples in this file to set the level chunks? 

https://github.com/google-research/arco-era5/blob/90f4c3dfc31692be73006e0ee841b620ecf81e7c/src/model-levels-to-zarr.py#L16

Reply:
Necessary changes were done.